.github/workflows/main--lint-unit-build-and-publish-images.yml

```diff
@@ -121,13 +121,6 @@ jobs:
         # if you change this, please remember to apply the same changes to branch--lint-unit-and-smoke-test.yml
         # so main branch cache can be shared with branches
         run: echo "DAYOFYEAR=$(date +%j)" >> $GITHUB_ENV
-      - name: Docker layer caching
```

Comment:
This is in place to fix the pipeline cache when it fills up and causes a out of memory failure